src/cavendish_particle_tracks/_analysis.py

```diff
@@ -19,6 +19,8 @@
     "A": [-15.00, 8.68],
 }  # cm
 
+VIEW_NAMES = ["Load data", "View1", "View2", "View3"]
```

Comment:
Is there a way to make `"Load data"` be a title? Because it's currently a selectable entry in your drop down menu... but actually the only options are View 1, ...

Reply:
Follows the same logic as New particle.... Sticking with it for the moment.